contracts/market/HardCoded.t.sol

```diff
@@ -52,7 +52,7 @@ contract HardCodedTest is FloatTest {
     assertEq(market.get_pool_value(IMarketCommon.PoolType.SHORT, poolIndexS2), 100e18, "Short pool 2 seed value is incorrect");
```

Comment:
Changes to tests based on changes to funding rate changes and max percent changes.

Reply:
Test code (ends in `*.t.sol` ) - not production code.